app/move/fields/create.js

```diff
@@ -336,6 +336,40 @@ module.exports = {
       text: 'fields::documents.hint',
     },
   },
+  move_agreed: {
+    validate: 'required',
+    component: 'govukRadios',
+    name: 'move_agreed',
+    fieldset: {
+      legend: {
+        text: 'fields::move_agreed.label',
+        classes: 'govuk-visually-hidden govuk-fieldset__legend--m',
+      },
+    },
+    items: [
+      {
+        id: 'move_agreed',
+        value: 'yes',
```

Comment:
This is probably fine for now, but we may want to update the values to `true`/`false` as that value will be sent directly to the API and will likely be how we store it rather than `yes`/`no`.